dashwellviz/figures.py

```diff
@@ -112,9 +121,10 @@ def make_composite_log(
     columns = []
     log = WellLog(n_tracks=n_tracks)
     for i, column_names in enumerate(lines):
+        log.update_subplot_titles({i: " ".join(column_names)})
```

Comment:
It might be a good idea to join the trace titles with a comma, as some traces will have names containing spaces already, and therefore the combination might become confusing unless there is another delimiter.

Reply:
@kinverarity1 Sounds reasonable!